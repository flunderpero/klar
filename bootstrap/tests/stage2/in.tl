let s = "Hi there"
if "the" in s then
    print("OK: 'the' has been found")
end
if "not" in s then
    print("ERR: 'not' has been found")
end
