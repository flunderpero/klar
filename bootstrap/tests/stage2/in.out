OK: 'the' has been found
